polaris/evaluate/_metric.py

```diff
@@ -30,6 +31,28 @@ def spearman(y_true: np.ndarray, y_pred: np.ndarray):
     return stats.spearmanr(y_true, y_pred).statistic
 
 
+def absolute_average_fold_error(y_true: np.ndarray, y_pred: np.ndarray):
+    """
+    Calculate the Absolute Average Fold Error (AAFE) metric.
```

Comment:
Could you explain what this metric measures? A brief description + a citation would be enough!